contracts/modules/DecentHatsCreationModule.sol

```diff
@@ -62,49 +62,41 @@ contract DecentHatsCreationModule is DecentHatsUtils {
      * We also make use of `KeyValuePairs` to associate the topHatId with the Safe.
      */
     function createAndDeclareTree(CreateTreeParams calldata params) external {
-        IHats hatsProtocol = params.hatsProtocol;
-        address hatsAccountImplementation = params.hatsAccountImplementation;
-        IERC6551Registry erc6551Registry = params.erc6551Registry;
-
         // Create Top Hat
         (uint256 topHatId, address topHatAccount) = _processTopHat(
-            hatsProtocol,
-            erc6551Registry,
-            hatsAccountImplementation,
+            params.hatsProtocol,
```

Comment:
I did...

Reply:
Honestly like, idk, it looked cleaner. I know that's not the best reason but I appreciate you calling me out because now I should actually do a gas profile and see how much extra cost this is.